### PR TITLE
feat: cypress with csr

### PR DESCRIPTION
**Pull request by estherthetesteramido**
#### 📲 What

A description of the change.

#### 🤔 Why
		
Cypress is cool.
		
#### 🛠 How
		
Setup base config. Tests are failing due to functionality awaiting.

#### 🕵️ How to test

`npm run test:cypress`

#### ✅ Acceptance criteria Checklist

- [x] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [x] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [x] Rebased/merged with latest changes from development and re-tested?
- [x] Meeting the Coding Standards?

### CODE REVIEW

File: packages/scaffolding-cli/templates/src/csr/README.md [hunk 1]
```diff
@@ -47,3 +47,68 @@ https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
...
+
+## Getting started
+
+Run `npm install --save-dev cypress axe-core cypress-axe @types/cypress-axe @applitools/eyes-cypress`
+
+## Folder structure
+
+├── __tests__
+│   └── fixtures
+│     ├── add-menu-response.json
```
Comment (naren-bellala) [inline packages/scaffolding-cli/templates/src/csr/README.md:62]: Shouldn't these be generic names?

File: packages/scaffolding-cli/templates/src/csr/package.json [hunk 2]
```diff
@@ -77,8 +82,8 @@
     "eslint-plugin-react-hooks": "^2.5.0",
     "eslint-plugin-security": "^1.4.0",
     "prettier": "^2.0.2",
-    "typescript": "^3.7.5",
-    "@svgr/webpack": "4.3.3"
```
Comment (naren-bellala) [inline packages/scaffolding-cli/templates/src/csr/package.json:81]: @svgr is useful lib for optimizing the svgs in CSR

File: packages/scaffolding-cli/templates/src/csr/README.md (hunk 1 above)
Comment (estherthetesteramido) [reply packages/scaffolding-cli/templates/src/csr/README.md:62]: Why do you say that? these fixures are stubbing out the menu calls made by the app.

Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/csr/README.md:62]: I get it. But these are meant to be abstract right. Not a big thing.

Comment (estherthetesteramido) [reply packages/scaffolding-cli/templates/src/csr/README.md:62]: You can't abstract stub fixtures when the the webapp is expecting it?

Comment (estherthetesteramido) [reply packages/scaffolding-cli/templates/src/csr/README.md:62]: Done in PR https://github.com/amido/stacks-webapp-template/pull/160

**Pull request merged**